### ISSUE TITLE
[Feature Request][Android] Target Android SDK Version 33

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#2537]
## Feature Description
Starting August 31, 2023 all new Android applications need to be target API level 33 to be submitted to Google Play.
AIR should built with target Android SDK Version 33.

## Known Workarounds
none

**Comment by neutronized** [comment]
same here, would be great if AIR builds with target Android SDK Version 33. When can this be expected? Thanks

**Comment by sjabberwocky** [comment]
@ajwfrost Hi Andrew! Any plans for Android 33 support?

Thanks for the AIR SDK update with iOS 16.
It's a great feeling when you just build the iOS version of the game and it works without any problems!

Currently I investigate this issue with Android build with latest AIR:
![изображение](https://user-images.githubusercontent.com/4690432/235425755-a12867dd-a473-4658-bd7c-c7960c9b2eb1.png)

**Comment by ajwfrost** [comment]
Hi

We'll be updating our builds to Android 33 shortly although as mentioned above, you can already override the target SDK level via the app descriptor file..

I'm a little concerned by the above vulnerability report and "unsafe" encryption - do you have any more details? It looks like these may be in some obfuscated code though, not sure whether you'd get these with a normal AIR app? but let us know if there's any updates we may need to make there..

thanks

**Comment by itlancer** [comment]
Latest AIR 50.2.3.2 uses Android SDK 33.
Thanks!

**Comment by Jasenvm** [comment]
Hi All

Not sure if this is a related issue.

I've got an interesting issue targeting API Level 33 while using SDK 50.2.5.1 on Google Play Internal Testing.

Google Play insists I'm not targeting API level 33, even though the Release Summary says I am.

<img width="1472" alt="image" src="https://github.com/airsdk/Adobe-Runtime-Support/assets/66523801/fe270a39-e63e-4cdc-a806-9d12410d30fd">

referencing this Policy:

<img width="906" alt="image" src="https://github.com/airsdk/Adobe-Runtime-Support/assets/66523801/acc3af27-86b3-4962-b658-eef4bc15a1c9">

I also uploaded a basic vanilla app with no ANEs to check, same Policy Violation warning.

Any ideas, what other information can I give to help understand the problem better?

Cheers
